cmd/schema.go

```diff
@@ -0,0 +1,460 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/odpf/salt/printer"
+	"github.com/odpf/salt/term"
+	stencilv1beta1 "github.com/odpf/stencil/server/odpf/stencil/v1beta1"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+)
+
+func SchemaCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "schema",
+		Aliases: []string{"schema"},
+		Short:   "Manage schema",
+		Long: heredoc.Doc(`
+			Work with schemas.
+		`),
+		Example: heredoc.Doc(`
+			$ stencil schema list
+			$ stencil schema create
+			$ stencil schema get
+			$ stencil schema update
+			$ stencil schema delete
+			$ stencil schema version
+		`),
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+	}
+
+	cmd.AddCommand(createSchemaCmd())
+	cmd.AddCommand(listSchemaCmd())
+	cmd.AddCommand(getSchemaCmd())
+	cmd.AddCommand(updateSchemaCmd())
+	cmd.AddCommand(deleteSchemaCmd())
+	cmd.AddCommand(versionSchemaCmd())
+
+	return cmd
+}
+
+func listSchemaCmd() *cobra.Command {
+	var host string
+	var req stencilv1beta1.ListSchemasRequest
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "list all schemas",
+		Args:  cobra.ExactArgs(1),
+		Example: heredoc.Doc(`
+		$ stencil schema list <schema-id>
+	`),
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			s := term.Spin("Fetching schema list")
```

Comment:
@ishanarya0 There is a slight change in this API for the spinner. Check guardian for reference. Also we can avoid label since this is only action and we are not changing any state.